lib/file_loader.js

```diff
@@ -77,12 +77,16 @@ HttpFileLoader.prototype = _.extend(new FileLoader(), {
   copy: function(suffix, destFile, cb) {
     var srcFile = url.resolve(this.prefix, suffix);
     fsUtil.ensureDirectoryExistsFor(destFile);
-    request(srcFile, opts, function (error, response, body) {
-      if (error || response.statusCode !== 200) {
-        return cb(error || new Error('Unexpected status code: ' + response.statusCode + ' copying ' + srcFile));
+    var r = request(srcFile, opts).on('error', function(error){ return cb(error);}).pipe(fs.createWriteStream(destFile));
```

Comment:
Piping the response shouldn't happen until the `statusCode` is checked. It should happen in the `on.('response'` handler.